src/Psalm/Codebase.php

```diff
@@ -2036,26 +2036,6 @@ public function getCompletionItemsForClassishThing(
         return $completion_items;
     }
 
-    /**
-     * @param list<CompletionItem> $items
-     * @return list<CompletionItem>
-     */
-    public function filterCompletionItemsByBeginLiteralPart(array $items, string $literal_part): array
```

Comment:
You'll need to keep the method to satisfy backward compatibility checks, but mark it `@deprecated to be removed in Psalm 6`.

Reply:
Thanks, I'll fix it. I didn't notice that a release had already been made. I thought this method was not yet in release.

Reply:
> PossiblyUnusedMethod: Cannot find any calls to method Psalm\Codebase::filterCompletionItemsByBeginLiteralPart

another problem )))

Reply:
problem is fixed by annotation `@api`...